concepticondata/conceptlists/RonaTas-2011-431.tsv

```diff
@@ -0,0 +1,431 @@
+ID	NUMBER	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS
+RonaTas-2011-431-1	1	carpenter	1042	CARPENTER
+RonaTas-2011-431-2	2	cannon, catapult	3202	CANNON
+RonaTas-2011-431-3	3	to fondle, pet, caress
+RonaTas-2011-431-4	4	stream, current	2935	CURRENT (STREAM)
+RonaTas-2011-431-5	5	false, imitation	3069	FALSE
+RonaTas-2011-431-6	6	pied (of an animals coat)		
+RonaTas-2011-431-7	7	apple	1320	APPLE
+RonaTas-2011-431-8	8	barrel (wooden container andstandard quantity)	2548	BARREL
```

Comment:
Ah, well spotted. Apparently I haven't accounted for line breakes in my pdf-parser because most of the meanings were short enough to be on the same line, so I didn't detect this shortcoming.